extensions/prototypes/labgraph_monitor/yarn.lock

```diff
@@ -0,0 +1,9601 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do we need to have this file?

Reply:
yarn unlike npm provides a deterministic approach to set up node_modules using Lockfiles that ensures that every install results in the exact same file structure in node_modules across all machines. Deleting **yarn.lock** may create errors when trying to set up the dependencies. ref: https://engineering.fb.com/2016/10/11/web/yarn-a-new-package-manager-for-javascript/